### PR TITLE
Change github URL to GITHUB_API_URL

### PR DESCRIPTION
**Pull request by dschunack**
Hi @pemtajo ,

I change the code a little bit to use the GITHUB API URL env variable instead of the hard coded github.com URL.
This allows to run the action on different github instances like github.com or an GitHUB Enterprise Instance.

### CODE REVIEW

File: settings.py [hunk 1]
```diff
@@ -5,6 +5,7 @@
 
 REPOSITORY = os.getenv("INPUT_REPOSITORY")
 GH_TOKEN = os.getenv("INPUT_GH_TOKEN")
+GH_API_URL = os.getenv("GITHUB_API_URL")
```
Comment (pemtajo) [inline settings.py:8]: Add in the file `action.yml` this property not required, with a default value to not break for anyone that not define this var.

Comment (pemtajo) [inline settings.py:8]: Update the README, adding this new env var in the docs

Comment (pemtajo) [inline settings.py:8]: Update the tests with this new env var, probably just adding the `github.com` in the mocks, don't need to add new ones if you don't want. Any problems with the tests sent a message that I help you

**Review by pemtajo — CHANGES_REQUESTED** [review]
Hi @dschunack, this is an awesome idea, I didn't think to use corporate Github. Very good

Just added some comments to increase your PR value. Any questions just ask me

**Comment by dschunack** [comment]
Hi  @pemtajo,
the GITHUB_API_URL env. var is a default variable and automatically added to each action. I think it's not needed to add it. It's automatically set by the github action based on the used github.

[github-context](https://docs.github.com/en/actions/learn-github-actions/contexts#github-context)

File: settings.py (hunk 1 above)
Comment (dschunack) [reply settings.py:8]: Hi, the GITHUB_API_URL env. var is a default variable and automatically added to each action. I think it's not needed to add it. It's automatically set by the github action based on the used github.

Comment (dschunack) [reply settings.py:8]: Hi, the GITHUB_API_URL env. var is a default variable and automatically added to each action. I think it's not needed to add it. It's automatically set by the github action based on the used github.

Comment (dschunack) [reply settings.py:8]: Hi, the GITHUB_API_URL env. var is a default variable and automatically added to each action. I think it's not needed to add it. It's automatically set by the github action based on the used github.

Comment (pemtajo) [reply settings.py:8]: Forget the var import
image

Comment (pemtajo) [reply settings.py:8]: Same problem as before, if you don't put it in the docs, you've implemented a cool feature that no one will know

**Review by pemtajo — APPROVED** [review]
I added some feedback comments for one next iteration. I approved the PR and created another one with the missing pieces

**Pull request merged**